plugins/inputs/t128_graphql/t128_graphql.go

```diff
@@ -189,13 +189,12 @@ func (plugin *T128GraphQL) Gather(acc telegraf.Accumulator) error {
 				}
 			}
 		}
-		return nil
```

Comment:
This is the real change I assume then. This goes from just happily reporting no errors to the errors being reported below so that Telegraf complains?

Reply:
yup. If there are errors, the collector won't return early and will still look for data. I'm about to push up some more changes for the new deadline header too though